apps/showcase/src/app/placeholder/placeholder.style.scss

```diff
@@ -0,0 +1,3 @@
+o3r-placeholder {
```

Comment:
Should we keep this file?

Reply:
As discussed we kept it for the others pages, we can removed them in another PR if you think it can be better without